src/emulator/dataconnectEmulator.ts

```diff
@@ -25,6 +26,9 @@ export interface DataConnectEmulatorArgs {
   auto_download?: boolean;
   rc: RC;
   config: Config;
+  autostartPostgres: boolean;
```

Comment:
Note: let's switch this this autoconnectToPostgres, so that it meshes with the DataConnectToolkitRegistry harold is working on.